src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs

```diff
@@ -98,15 +98,7 @@ public bool Render(HtmlRenderer renderer, MarkdownObject obj, Action<string> log
 
             var htmlAttributes = new HtmlAttributes();
 
-            // alt is allowed to be empty for icon type image
-            if (string.IsNullOrEmpty(alt) && currentType == "icon")
-            {
-                htmlAttributes.AddProperty("src", _context.GetLink(src, obj));
-            }
-            else
-            {
-                htmlAttributes.AddProperty("src", _context.GetImageLink(src, obj, alt));
-            }
+            htmlAttributes.AddProperty("src", _context.GetImageLink(src, obj, alt, currentType == "icon"));
```

Comment:
Consider pass the image type here in case we need to process image type differently in the future.
```suggestion
            htmlAttributes.AddProperty("src", _context.GetImageLink(src, obj, alt, currentType));
```